modules/dop/endpoints/workbench.go

```diff
@@ -130,49 +133,56 @@ func (e *Endpoints) GetWorkbenchData(ctx context.Context, r *http.Request, vars
 		issueItem.IssueList = issues
 		issueItem.ProjectDTO = v
 
+		var wg sync.WaitGroup
+		wg.Add(len(expireDays))
 		for index, et := range expireDays {
-			etIssueReq := apistructs.IssuePagingRequest{}
-			etIssueReq.OrgID = int64(workReq.OrgID)
-			etIssueReq.StartFinishedAt = timeList[index][0] * 1000
-			if et == "unspecified" {
-				etIssueReq.IsEmptyPlanFinishedAt = true
-			} else {
-				if timeList[index][1] != 0 {
-					etIssueReq.EndFinishedAt = timeList[index][1] * 1000
+			go func(idx int, ed string) {
+				defer wg.Done()
+				etIssueReq := apistructs.IssuePagingRequest{}
+				etIssueReq.OrgID = int64(workReq.OrgID)
+				etIssueReq.StartFinishedAt = timeList[idx][0] * 1000
+				if et == "unspecified" {
+					etIssueReq.IsEmptyPlanFinishedAt = true
+				} else {
```

Comment:
it's better package this go func() as a real function, and give some unit test

Reply:
it`s hard to package this go func(), because Contextual connection